schemas/test/en/test_progress_value_source_blocks_chained.json

```diff
@@ -0,0 +1,116 @@
+{
```

Comment:
What is the behaviour different between this and `test_progress_value_source_blocks`, they seem to prove the same thing?. This file doesn't seem to be used in any tests either?. Is it needed?

Reply:
`blocks_chained` was indeed redundant, removed